packages/cdk8s-cli/snapshotResolver.js

```diff
@@ -0,0 +1,15 @@
+const {basename, dirname, join} = require('path');
+const bazelDir = process.env['BUILD_WORKSPACE_DIRECTORY'] || process.cwd();
+const bazelRoot = process.env['BAZEL_PATCH_ROOT'] || '';
+const binDir = 'bin';
+const pathEnd = dirname(bazelRoot.slice(bazelRoot.lastIndexOf(binDir) + binDir.length));
```

Comment:
HAA?A?

Reply:
I'll add a comment to the header of this file, but basically we need to pierce of the veil of the Bazel sandbox in order to update snapshots. Otherwise the updated snapshots would just exist in the sandbox and never get moved back to the source. See earlier comment about why this functionality was added to Jest in the first place.

Reply:
ts-jest you said, no?